### PR TITLE
implemented hero-section

### PR DESCRIPTION
**Pull request by abygeorge7**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -45,7 +45,34 @@ <h1 class="navbar-dropdown-item">
             </div>
         </div>
     </nav>
-    
+
+    <section class="top-hero-container">
```
Comment (BenjaminSao) [inline index.html:49]: The format is usually you have a section and within the section you have a container therefore, this should be named top-hero-section and inside you have the top-hero-container.

File: index.html [hunk 2]
```diff
@@ -45,7 +45,34 @@ <h1 class="navbar-dropdown-item">
...
+        <div class="hero-content">
+            <div>
```
Comment (BenjaminSao) [inline index.html:51]: This div has no classes. Is there a reason it's here?

File: index.html [hunk 3]
```diff
@@ -45,7 +45,34 @@ <h1 class="navbar-dropdown-item">
...
+                <div>
```
Comment (BenjaminSao) [inline index.html:52]: Unnecessary div

File: index.html [hunk 4]
```diff
@@ -45,7 +45,34 @@ <h1 class="navbar-dropdown-item">
...
+                    <h1 class="hero-heading">Build better processes</h1>
+                </div>
+                <div>
```
Comment (BenjaminSao) [inline index.html:55]: Again... Unnecessary div

File: index.html [hunk 5]
```diff
@@ -45,7 +45,34 @@ <h1 class="navbar-dropdown-item">
...
+                    <p class="hero-paragraph">Your customers won’t settle and neither should you. With Pluralsight,
+                        technology teams know more
+                        and
+                        work
+                        better together. Develop the skills you need for your top priorities. Improve workflows with
+                        actionable
+                        data. Build better—every time.
+                    </p>
+                </div>
+                <div class="hero-button-container">
```
Comment (BenjaminSao) [inline index.html:65]: buttons

File: index.html [hunk 6]
```diff
@@ -45,7 +45,34 @@ <h1 class="navbar-dropdown-item">
...
+                    <div class="hero-button-size">
```
Comment (BenjaminSao) [inline index.html:66]: Naming could be a bit better like hero-buttons. If the button sized is reused you can have like a large-button class...

File: style.css [hunk 7]
```diff
@@ -121,4 +121,125 @@ h1
...
+  height: 56.36px;
+}
+
+.try-for-free-button:hover 
+{
+  /*horizontal(offset x) zero vertical(offset y) zero blur radius zero and spread radius set to 1px*/
+  box-shadow: inset 0 0 0 1px #fff;
+}
+
+@media screen and (max-width: 740px)
```
Comment (BenjaminSao) [inline style.css:214]: Use min-width for a more mobile approach

File: style.css [hunk 8]
```diff
@@ -121,4 +121,125 @@ h1
   {
     margin-right: 10px;
   }
+
+.top-hero-container
```
Comment (BenjaminSao) [inline style.css:125]: Can you make sure you are using flex-box properly?

**Review by BenjaminSao — CHANGES_REQUESTED** [review]

File: index.html (hunk 4 above)
Comment (abygeorge7) [reply index.html:55]: deleted

File: index.html (hunk 6 above)
Comment (abygeorge7) [reply index.html:66]: the above class just for display change to column.The button name is try-for-free-button and hope ok

File: style.css (hunk 8 above)
Comment (abygeorge7) [reply style.css:125]: i changed to auto

**Pull request closed without merging**